src/internal/clusterstate/v2.6.0/pfsdb.go

```diff
@@ -174,6 +175,7 @@ func removeAliasCommits(ctx context.Context, tx *pachsql.Tx) error {
 	realAncestors := make(map[string]*v2_5_0.CommitInfo)
 	childToNewParents := make(map[*pfs.Commit]*pfs.Commit)
 	for _, ci := range deleteCommits {
+		log.Info(ctx, "computing parent / children after deleted alias commit", zap.String("commit", oldCommitKey(ci.Commit)))
```

Comment:
WDYT about adding a total (+ counter) for the index of the item processed?

Reply:
I added a progress field for each log that occurs in a for loop. It contains the string "index / total".